templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -204,13 +204,14 @@ Parameters:
     ConstraintDescription: >-
       Minimum length of 1. Maximum length of 100. Must start with a letter or
       number.
+    Default: cluster-eks-tap
```

Comment:
nit: the object being created is an eks cluster, so this default name seems redundant. I'd be more in favor of something like `tanzu-application-platform`

Reply:
I hear you.  Sometimes it is very hard to know the AWS resources just from names. We need to add TAGs to remember this. I have added this default to lower one more input parameter from the user & have a short name to use in CLIs. I shall change this name in the future if more questions around this naming.